### DISCUSSION TITLE
Additional rewards for documenting the solving of a bounty.

### DISCUSSION THREAD

**Discussion by rahimklaber** [kalepail/stellar-quest-bounties#74] [Bounty Ideas]
# Bounty idea
It would be cool if for some bounties additional rewards could be given if the process of solving the bounty is documented in some way. It could be an indepth tutorial, a more general overview of how the bounty was solved or even an educational video.

## Things to think about
* If multiple people are doing the same bounty, should we prioritize those who document the process?
* Maybe the documenting part could be a seperate bounty so that hunters can still get rewarded if their bounty did not get chosen as the solution.

**Comment by hanseartic** [comment]
Great idea! I would split it. We could argue that solving *before* documenting leads to a bonus compared to just documenting.

So people who just create a tut will also have a chance but are encouraged to actually solve before doing so. I believe making a tut out of practical experience will lead to substantially better results than just writing up the theories.

**Reply by rahimklaber** [reply]
Thats a good idea, I like it.

**Comment by kalepail** [comment]
I like this, though I'm not sure how we'd structure it. Which bounties are eligible? How many documentations do we need? Would these be separate bounties or "tack on" bonuses? Seems like we'd need to do some additional thinking around this. In general I feel like written bounties should be separate to the bounties themselves. With their own scope and prize etc. That way we can more fully control scope, reward amount and the number of entries we want.

I think I've seen a bounty for solving old Stellar Quest series. That sort of concept implemented here could be really nice.
